packages/server/src/appMigrations/tests/migrations.spec.ts

```diff
@@ -52,4 +53,26 @@ describe("migrations", () => {
       },
     })
   })
+
+  it("should disable migrations if any migration is disabled", () => {
+    // remove all migrations
+    migrations.MIGRATIONS.splice(0, migrations.MIGRATIONS.length)
```

Comment:
Modifying global state in a test is risky. I'd recommend one of two things:

1. Put this back how you found it when you're done.
2. Make `getLatestEnabledMigrationId()` accept a parameter that is the list of migrations to operate over, removing the need to modify global state.

I like 2 more than 1.